network/peer/peer.go

```diff
@@ -523,20 +523,12 @@ func (p *peer) writeMessages() {
 	}
 
 	myVersion := p.VersionCompatibility.Version()
-	legacyApplication := &version.Application{
-		Name:  version.LegacyAppName,
```

Comment:
Is `LegacyAppName` dead code now?

Reply:
Replaced it in tests with `Client` and removed the variable